extras/check_code_style.sh

```diff
@@ -229,7 +251,10 @@ if [ "$format_error_files" != "" ]; then
     echo "  $f"
   done
   echo
-  echo "Perform '$CLANG_FORMAT -i <filename>' on them, otherwise TravisCI will report failure!"
+  echo "TravisCI will report failure!"
+  echo "- On C/CPP files, perform '$CLANG_FORMAT -i <filename>' on them."
+  echo "- On Python files, perform 'pep8ify -w <filename>' on them and"
```

Comment:
Please remove the suffix "on them". See [here](http://uncyclopedia.wikia.com/wiki/Redundancy).